cli/contract_test.go

```diff
@@ -58,12 +58,13 @@ func TestCalcHash(t *testing.T) {
 			"--in", "./testdata/verify.nef123", "--manifest", manifestPath)...)
 	})
 	t.Run("invalid file", func(t *testing.T) {
-		p := path.Join(os.TempDir(), "neogo.calchash.verify.nef")
+		p, err := ioutil.TempFile("", "neogo.calchash.verify.nef")
```

Comment:
It is worth to use `neogo.calchash.verify.*.nef` pattern (and later) because `Cleanup` doesn't always execute when tests are interrupted with `Ctrl+C`. This can lead to some annoying test fails on the next invocation.

Reply:
> The filename is generated by taking pattern and adding a random string to the end. If pattern includes a "*", the random string replaces the last "*".

https://pkg.go.dev/io/ioutil#TempFile